### DISCUSSION TITLE
新UI，以及新功能的想法

### DISCUSSION THREAD

**Discussion by MLAcookie** [MLAcookie/MAA1999WPF#21] [General]
最近事情多（摸鱼也多），所以这项目稍微搁置了一会儿
这几天抽空研究了下HandyControl，然后整了个UI
发布在dev-HandyControl分支下
目前这个分支只是搭了个框架，功能还没写

现在的问题是UI的布局可能比较抽象（不知道怎么设计好看）
然后就是因为HandyControl没有提供NavigationView的控件，我只能用SideMenu来手搓一个类似的
或许有更好的实现方式？（或者不用类似NavigationView的菜单，但我确实不大会设计）

然后就是新功能这块，目前打算是支持多config切换
然后再写一个定时执行的功能（UI的框架也是这样搭的）

**Comment by KaronGH** [comment]
(这段时间忙炸了没看到新的分支)
我看到这个dev分支把dotnet提到了8.0版本，如果是确认以后用8.0版本开发的话我本地这边就基于这个分支修改了
另外我觉得如果有做VersionHelper类似的功能的话，构建时自动下载MAA-BIN的功能似乎可以暂时注释掉，因为一是二进制下载可以放到RUN的时候做，二是把下载部分的代码写到.csproj里面似乎不太方便指定proxy？（这块我不是很懂，我这里经常因为鄙司防火墙build失败）

**Reply by MLAcookie** [reply]
确实是可以注释掉了，这个随意吧
